src/Resources/ShipmentOptions.php

```diff
@@ -31,6 +31,9 @@ class ShipmentOptions extends BaseResource
     /** @var bool */
     public $signature;
 
+    /** @var array|null */
+    public $insurance;
```

Comment:
I think it may be better to make this attribute an empty array by default.
And set it like so in the `setDefaultOptions` method.

Then filter that out in. the `toArray` option.

Reason the `setDfaultOptjons` method is also used to "reset" to the default options, for example when calling the `mailboxpackage` method on a `Parcel` object.